pkg/armhelpers/azurestack/disk_test.go

```diff
@@ -9,7 +9,10 @@ import (
 )
 
 func TestDeleteManagedDisk(t *testing.T) {
-	mc := NewHTTPMockClient()
+	mc, err := NewHTTPMockClient()
+	if err != nil {
+		t.Fatalf("failed to create HttpMockClient - %s", err)
```

Comment:
Should these be `t.Error()` statements instead of `t.Fatalf()` statements?

Reply:
The testing package's `Fatalf` doesn't involve `os.Exit`, so no worries there:
> Fatalf is equivalent to Logf followed by FailNow.